core/modes/sleep_mode.talon

```diff
@@ -7,3 +7,5 @@ settings():
     user.mouse_enable_pop_click = 0
     # Stop mouse scroll down using hiss noise
     user.mouse_enable_hiss_scroll = 0
+    # Enable pop wake
+    user.mouse_enable_pop_wake = 1
```

Comment:
Just noticed this. This should probably default to false; popping can occasionally have false positives so for something that can have as many side effects as waking Talon it should probably be aped in, but we can advertise it in the README as a setting that people can enable.

Also added a note that this will intrinsically disable popping to click, since you can't have both.

```suggestion
    # Enable pop wake. Note that if this is enabled, it will disable popping to click in sleep mode, if you set `mouse_enable_pop_wake = 2`.
    user.mouse_enable_pop_wake = 0
```